src/components/Avatar/Avatar.component.tsx

```diff
@@ -272,23 +264,23 @@ const Avatar: FC<AvatarProps> = ({
       {background?.src && <Box {...background} />}
```

Comment:
Shadows no longer work, as the spotlights aren't set to cast shadows.
Also, turning on `castShadow` for one of the spotlights doesn't give the intended result.

<img width="1034" alt="Screenshot 2024-01-02 at 15 22 25" src="https://github.com/readyplayerme/visage/assets/14032500/eed30ffb-6e73-47a3-b625-29162b360d3a">

vs

<img width="897" alt="Screenshot 2024-01-02 at 15 22 44" src="https://github.com/readyplayerme/visage/assets/14032500/d5fdb4e9-ecc2-425e-98b6-0b4795d49b8a">

Reply:
The spotlights may not have the right rotation to reproduce the original shadow angle. A workaround could be to use a dummy directional light to cast the shadow in the desired direction without an intensity.

![Screenshot from 2024-01-02 14-33-05](https://github.com/readyplayerme/visage/assets/152877805/89b5c882-4f7b-4424-95c2-73e67681de7d)